default_organisation.bash

```diff
@@ -212,6 +213,25 @@ cmd_recrypt() {
 	git_commit "Recrypt $* for ${target_groups[*]}."
 }
 
+# Check for an extension update
+cmd_update() {
+	echo "Looking for update..."
+	tmp_file="/tmp/pass-group-$EXTENSION_NAME.bash"
+	curl -s https://raw.githubusercontent.com/erdnaxe/pass-group/master/default_organisation.bash > $tmp_file
```

Comment:
We should definitely be careful with curl options. Why not using `-o $tmp_file` ?
Also I do not agree on silent mode by default, it is good practice to show user the curl happening to be more transparent / make the user more patient.

Reply:
I also do not like to hardcode the URL here, but I think people still have more trust in a github URL than a custom server. Good enough

Reply:
I removed the silent option, and added the output option.
I don't know what to do with the download URL.